test/test.js

```diff
@@ -112,8 +112,8 @@ describe('loadParsedJson', () => {
 
   describe('with a path to a file that does not exist', () => {
 
-    it('should throw an error', () => {
-      loader.loadParsedJson.bind(null, '.', 'does-not-exist').should.throw(/ENOENT/);
+    it('should not throw an error', () => {
+      loader.loadParsedJson.bind(null, '.', 'does-not-exist').should.not.throw();
```

Comment:
You don't actually have to use `.should.not.throw()` at all. Not sure if that gives better error feedback, but just invoking it would be sufficient. Also removes the need for bind()